project/frontend/src/components/ResultsMap.js

```diff
@@ -30,12 +39,16 @@ function MapContainer(props) {
       const numOfMarkers = Math.min(restaurants.length, 4);
       for (let i = 0; i < numOfMarkers; i++) {
         const coords = restaurants[i].key.latLngCoords;
+        const numInList = (i + 1).toString();
```

Comment:
Do we need to cast this to a string?

Reply:
I use this value in strings so i'd rather just call toString once as opposed to every time I reference the variable!